src/backends/gpu/matrix_state_interleaved.hpp

```diff
@@ -0,0 +1,255 @@
+#pragma once
```

Comment:
Inconsistent commenting style in the source (capitalization, sentence fragments, blank lines etc.).